libflux/src/flux/lib.rs

```diff
@@ -603,10 +603,56 @@ mod tests {
     use core::parser::Parser;
     use core::semantic::convert::convert_file;
     use core::semantic::env::Environment;
+    use core::semantic::fresh::Fresher;
     use core::semantic::nodes::infer_file;
-    use core::semantic::types::MonoType;
+    use core::semantic::types::{MonoType, Row, Tvar, TvarMap};
     use core::{ast, semantic};
 
+    pub struct MonoTypeNormalizer {
```

Comment:
Can you add ` #[cfg(test)]` to this struct?

Reply:
The entire struct is inside a mod marked `#[cfg(test)]`

Reply:
That's great!